tests/addnode.py

```diff
@@ -20,23 +20,9 @@ def run(args):
     ) as network:
         primary, others = network.start_and_join(args)
 
-        LOG.debug("Adding a node conflicting with an existing node")
```

Comment:
This is now gone as we test this as a unit test. It is much better since `infra.ccf` now only exposes `create_and_trust_node()` to create a node that will guarantee that the `node_id` in the python is always in sync with the `node_id` given by the node in the KV.